### ISSUE TITLE
Build problems on Windows when downloading openfast/dev as a zip instead of a git clone

### ISSUE THREAD

**Issue by ghaymanNREL** [OpenFAST/openfast#84]
I receive fatal build error: 
Not a git repository (or any parent up to mount point /cygdrive/c)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

At this point I have not investigated any further.  Should test a non-Windows cmake build under the same conditions: zip download instead of git clone

**Comment by rafmudaf** [comment]
@ghaymanNREL were you trying to only build openfast or the testing framework, as well?

**Comment by ghaymanNREL** [comment]
@rafmudaf, only openfast.  This is a Windows, vs-build system, issue that is tied to the custom prebuild step which uses the script: CreateGitVersion.bat.  That script in turns attempts a git describe, which does not execute because it is not a git repository.

**Comment by bjonkman** [comment]
You'd also get this error if you had a git clone but don't have the git-for-Windows command-line integration installed.

**Comment by rafmudaf** [comment]
On unix, this isn't an error in the build process but you do get a binary without a version:

```
mbp@~/Downloads/openfast-dev/install/bin: ./openfast 

 **************************************************************************************************
 OpenFAST

 Copyright (C)  National Renewable Energy Laboratory

 This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
 See the "LICENSE" file distributed with this software for details.
 **************************************************************************************************

 OpenFAST-HEAD-HASH-NOTFOUND
```

And the testing framework will not work since the git submodules can't be downloaded.

**Comment by rafmudaf** [comment]
@ghaymanNREL Is this an issue that we should fix? Since the GitHub zip download feature does not include the `.git` directory, its distributing an unversioned openfast.

If that's deemed acceptable, then the build scripts for all platforms could set the version string to "UNVERSIONED" when necessary.

@bjonkman brings up a good point which the suggestion above does not fix. Sharing a legitimately versioned copy of openfast does not guarantee a versioned build. I think we need to add a requirement for git in the [required software section](http://openfast.readthedocs.io/en/master/source/install/install_cmake_linux.html#required-software-for-building-openfast) of the documentation for all platforms.

**Comment by jjonkman** [comment]
Fixed by #194.
